src/ring_theory/graded_algebra/homogeneous_ideal.lean

```diff
@@ -423,13 +451,13 @@ begin
 end
 
 @[simp] lemma homogeneous_ideal.homogeneous_hull_coe_eq_self (I : homogeneous_ideal 𝒜) :
-  (I : ideal A).homogeneous_hull 𝒜 = I :=
-subtype.coe_injective $ ideal.is_homogeneous.homogeneous_hull_eq_self I.prop
+  I.to_ideal.homogeneous_hull 𝒜 = I :=
+by ext1; convert ideal.is_homogeneous.homogeneous_hull_eq_self I.is_homogeneous
```

Comment:
You should be able to use `homogeneous_ideal.to_ideal_injective` instead of `by ext1; convert`.

Reply:
done